lib/axlsx/workbook/worksheet/pivot_table.rb

```diff
@@ -193,19 +203,28 @@ def to_xml_string(str = '')
       unless pages.empty?
         str << ('<pageFields count="' << pages.size.to_s << '">')
         pages.each do |page_value|
-          str << ('<pageField fld="' << header_index_of(page_value).to_s << '"/>')
+          str << ('<pageField fld="' << header_index_of(page_value).to_s << '" hier="-1"/>')
         end
         str << '</pageFields>'
       end
       unless data.empty?
         str << "<dataFields count=\"#{data.size}\">"
         data.each do |datum_value|
-          str << "<dataField name='#{@subtotal} of #{datum_value[:ref]}' fld='#{header_index_of(datum_value[:ref])}' baseField='0' baseItem='0'"
-          str << " subtotal='#{datum_value[:subtotal]}' " if datum_value[:subtotal]
+          # The correct name prefix in ["Sum","Average", etc...]
+          str << "<dataField name=\"#{(datum_value[:subtotal]||'').titleize} of #{datum_value[:ref]}\" fld=\"#{header_index_of(datum_value[:ref])}\" baseField=\"0\" baseItem=\"0\""
+          str << " subtotal=\"#{datum_value[:subtotal]}\" " if datum_value[:subtotal]
           str << "/>"
         end
         str << '</dataFields>'
       end
+      # custom style
+      if style_info.present?
```

Comment:
`Hash#present?` is not defined when running the tests.

ActiveSupport is _not_ available as a depdendency.